File: php/Member.php
<?php
class Member {

    // 프로퍼티(멤버 변수)
	private $memName;
    private $memID;
    private $memPW;
    private $memEmail;
    private $memSsn;
	
    // 메소드
	public function getMemName() {
        return $memName;
    }
    public function setMemName($memName) {
        $this->memName = memName;
    }
    public function getMemID() {
        return $memID;
    }
    public function setMemID($memID) {
        $this->memID = memID;
    }
    public function getMemPW() {
        return $memPW;
    }
    public function setMemPW($memPW) {
        $this->memPW = memPW;
    }
    public function getMemEmail() {
        return memEmail;
    }
    public function setMemEmail($memEmail) {
        $this->memEmail = memEmail;
    }
    public function getMemSsn() {
        return $memSsn;
    }
    public function setMemSsn($memSsn) {
        $this->memSsn = memSsn;
    }
}
?>

File: app/src/main/java/kr/hongik/mnms/Member.java
package kr.hongik.mnms;

import java.io.Serializable;

public class Member implements Serializable {
    private static final long serialVersionUID = 1L;

    private String memName;
    private String memID;
    private String memPW;
    private String memEmail;
    private String memSsn; //주민번호
    private String accountNum;
    private boolean isChecked;
    private String ip;

    //친구 그룹

    public String getIp() {
        return ip;
    }

    public void setIp(String ip) {
        this.ip = ip;
    }

    public String getMemName() {
        return memName;
    }

    public void setMemName(String memName) {
        this.memName = memName;
    }

    public String getMemID() {
        return memID;
    }

    public void setMemID(String memID) {
        this.memID = memID;
    }

    public String getMemPW() {
        return memPW;
    }

    public void setMemPW(String memPW) {
        this.memPW = memPW;
    }

    public String getMemEmail() {
        return memEmail;
    }

    public void setMemEmail(String memEmail) {
        this.memEmail = memEmail;
    }

    public String getMemSsn() {
        return memSsn;
    }

    public void setMemSsn(String memSsn) {
        this.memSsn = memSsn;
    }

    public String getAccountNum() {
        return accountNum;
    }

    public void setAccountNum(String accountNum) {
        this.accountNum = accountNum;
    }

    public boolean isChecked() {
        return isChecked;
    }

    public void setChecked(boolean checked) {
        isChecked = checked;
    }
}
